ui/src/screens/SearchScreen/SearchScreen.jsx

```diff
@@ -52,14 +52,6 @@ const messages = defineMessages({
     id: 'search.facets.facet.addresses',
     defaultMessage: 'Addresses',
   },
-  facet_mime_type: {
```

Comment:
There's actually a `mimetypes` field on each document we could query. Author can go away.

Reply:
recovered